service/src/test/java/bio/terra/workspace/service/workspace/AwsCloudContextUnitTest.java

```diff
@@ -70,4 +70,14 @@ void serdesTest() {
         () -> AwsCloudContext.deserialize(makeDbCloudContext(CloudPlatform.AWS, junkJson)),
         "Junk JSON should throw");
   }
+
+  /* TODO(BENCH-319)
```

Comment:
if this is not used right now, this is just adding dead code. can you remove this and add it when the "change in library" is applied?

Reply:
Sure, np. The lib change is here - https://github.com/DataBiosphere/terra-aws-resource-discovery/pull/42/files
removed the comment here